modules/serverless-rn-1-35-1.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies
+//
+// * about/serverless-release-notes.adoc
+
+:_content-type: REFERENCE
+[id="serverless-rn-1-35-1_{context}"]
+= Red Hat {ServerlessProductName} 1.35.1
+
+{ServerlessProductName} 1.35.1 is now available. This release of {ServerlessProductName} addresses identified Common Vulnerabilities and Exposures (CVEs) to enhance security and reliability. Fixed issues and known issues that pertain to {ServerlessProductName} on {ocp-product-title} are included in the following notes:
+
+[id="fixed-issues-1-35-1_{context}"]
+== Fixed issues
+
+* Previously, the `kafka-controller-post-install-1.35` job failed after installing `KnativeKafka` if the `KafkaChannel` feature was not enabled. This issue has been fixed. The post-install job now completes successfully, even when `KafkaChannels` are disabled.
+
+* Previously, the `sinks.knative.dev` API group was missing from the KnativeEventing aggregate cluster roles. As a result, non-admin users encountered the following error when running `oc get all` command:
++
+[source,sql]
+----
+jobsinks.sinks.knative.dev is forbidden: User "user" cannot list resource "jobsinks" in API group "sinks.knative.dev" in the namespace "test"
+----
++
+This issue has been resolved. The `sinks.knative.dev` API group is now included in the `knative-eventing-namespaced` aggregate cluster roles. Users with `view`, `edit`, or `admin` roles can now read and list sinks in their own namespaces.
+
+[id="known-issues-1-35-1_{context}"]
+== Known issues
+
+* In some scenarios, cluster-scoped resources such as webhook configurations may remain after uninstalling, reinstalling, or upgrading KnativeServing or the {ServerlessOperatorName}. These leftover resources can block the reconciliation process and prevent KnativeServing from installing properly. When this issue occurs, you may see an error similar to the following:
++
+[source,terminal]
+----
+failed to apply non rbac manifest: Internal error occurred: failed calling webhook \"webhook.serving.knative.dev\": failed to call webhook: Post \"https://webhook.knative-serving.svc:443/?timeout=10s\": no endpoints available for service \"webhook\"
```

Comment:
Should there be a hyphen between "non" and "rbac"?

Reply:
I verified this with the SME, which is also present in the reference JIRA: [SRVKS-1301](https://issues.redhat.com/browse/SRVKS-1301). So, keeping it as it is.